sky/utils/schemas.py

```diff
@@ -83,6 +83,12 @@ def _get_single_resources_schema():
                     }
                 }],
             },
+            'labels': {
+                'type': 'object',
+                'additionalProperties': {
+                    'type': 'string'
+                }
+            },
```

Comment:
Ahh, actually, after thinking more, I found it might be more feasible in `any_of`, because the following case:

```yaml
resources:
  any_of:
    - cloud: aws
       labels:
           cloud_name: aws
    - cloud: gcp
       labels:
           cloud_name: gcp
```

Reply:
Hmm, good point. I think it might make sense in the example where different cost tracking labels need to be setup for different clouds. I'll add this and change semantics to update instead of overwrite.

Reply:
Added and tested with:
```
resources:
  labels:
    cloud_name: dontknow
    otherlabel: othervalue
  any_of:
    - cloud: gcp
      labels:
        cloud_name: gcp
    - cloud: kubernetes
      labels:
        cloud_name: kubernetes
```

Created pod on k8s has:
```
Labels:           cloud_name=kubernetes
                  otherlabel=othervalue
```